iOS/MSMapsModulesReferenceApp/MSMapsModulesReferenceApp/ViewController.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  ViewController.swift
+//  MSMapsModulesReferenceApp
+//
+//  Created by Elizabeth Bartusiak (t-elbart) on 6/24/20.
+//  Copyright © 2020 Microsoft Corporation.
+//  Licensed under the MIT license.
+//
+
+import MicrosoftMaps
+import MSMapsModules
+import UIKit
+
+class ViewController: UIViewController {
+    @IBOutlet var mMapView: MSMapView!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        /* Add your credentials key in the following line.
+         See
+         https://docs.microsoft.com/en-us/bingmaps/getting-started/bing-maps-dev-center-help/getting-a-bing-maps-key
+         for instructions on how to get a key.
+         */
+        mMapView.credentialsKey = "YOUR_KEY_HERE"
```

Comment:
This seems error prone. Is there a way to put this in a resource of some kind and add that file to .gitignore so this never gets accidentally pushed?